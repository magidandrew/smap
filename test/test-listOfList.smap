int main (){
   int i = 0;
   list list char x = [['g','h'],['h','e','l','l','o']];
   list list char y = ["hey"];
   println(x[0]);
   println(y[0]);
   println(x[0][0]);
   println(x[1][1]);
   for(i; i< 5; i= i+1){
       print(x[1][i]); 
   }
    return (0);
}