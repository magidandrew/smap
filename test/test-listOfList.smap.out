gh
hey
g
e
hello